src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -50,6 +50,12 @@ ucs_config_field_t uct_ib_mlx5_iface_config_table[] = {
      ucs_offsetof(uct_ib_mlx5_iface_config_t, mmio_mode),
      UCS_CONFIG_TYPE_ENUM(uct_ib_mlx5_mmio_modes)},
 
+    {"AR_ENABLE", "n",
+     "Enable Adaprive Routing (out of order) feature on SL that supports it. "
```

Comment:
`Adaptive`

Reply:
please add `\n`, otherwise line would be too long

Reply:
done